src/components/gui/gui.jsx

```diff
@@ -157,9 +156,6 @@ const GUIComponent = props => {
                 dir={isRtl ? 'rtl' : 'ltr'}
                 {...componentProps}
             >
-                {previewInfoVisible ? (
```

Comment:
You need to remove `previewInfoVisible` from the list above on line 110. That is one of the things causing the lint errors.